compiler/lib/php_from_rehp.re

```diff
@@ -379,7 +485,7 @@ let rec foldVars =
         let (out, initMapped) =
           optAppendOutput(curOut, mapper(curInput), eo);
         let out = {...out, dec: addOne(out.dec, id)};
-        let input = addOne(curInput, id);
+        let input = {vars: addOne(curInput.vars, id), label: curInput.label};
```

Comment:
Or you could use `...` spread so that it requires fewer changes if we ever add more fields.

```
let input = {...curInput, vars: addOne(curInput.vars, id)};
```